install/operator/deploy/operatorsource.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: syndesis
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: lgarciaac
```

Comment:
Maybe not add a personal one or add some comments

Reply:
I will remove this file